specification/entities.yaml

```diff
@@ -614,6 +614,8 @@ components:
         - KAFKA_SERVICE
         - VECTOR_STORE
         - RELATIONSHIP
```

Comment:
should we remove RELATIONSHIP if we have separate ENTITY_RELATIONSHIP and GRAPH_RELATIONSHIP?

Reply:
I was thinking about to remove it after we start using other two. But it seems we can do it painlessly right now